common/utils/filters.ts

```diff
@@ -115,3 +128,21 @@ export const getSelectedFilterColor = (form: HTMLFormElement): string => {
       : imagesColorValue;
   return imagesColor;
 };
+
+export const sortAggregationBucket = (
+  filter: CatalogueAggregationBucket[],
+  sortBy?: 'alphabetical' | 'count'
```

Comment:
`sortBy: 'alphabetical' | 'count' = 'count'`

It seems weird to offer a method that has the potential to do nothing.

Reply:
not all the aggregations atm return by count, which i thought we could use. Happy to remove this..